CONTRIBUTING.md

```diff
@@ -829,6 +830,21 @@ class Event(models.Model):
   a_field(..., default=lambda self: self._default_get())
   ```
 
+### Exceptions
+  The `pass` into block except is not a good practice!
+
+  By including the `pass` we assume that our algorithm can continue to function after the exception occurred
+
+  If you really need to use the `pass` consider logging that exception
+
+  ```python
+      try:
```

Comment:
Might want to reduce the indentation level by one, this is double indented in the example block right now